src/makeRobot.js

```diff
@@ -38,7 +38,51 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
```

Comment:
It's better to add empty lines between functions to make the code easier to read
```suggestion
    get location() {
      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
    },
    
    get info() {
      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
    },
```